mod_customized/controllers.py

```diff
@@ -0,0 +1,102 @@
+"""
+mod_customized Controllers
+===================
+In this module, users can test their fork branch with customized set of regression tests
+"""
+from flask import Blueprint, g, request, redirect, url_for
+from github import GitHub, ApiError
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+
+from decorators import template_renderer, get_menu_entries
+from mod_auth.controllers import login_required, check_access_rights
+from mod_auth.models import Role, User
+from mod_test.models import Fork, Test, TestType, TestPlatform
+from mod_customized.forms import TestForkForm
+from mod_customized.models import TestFork
+from mod_test.controllers import get_data_for_test, TestNotFoundException
+from mod_auth.controllers import fetch_username_from_token
+from sqlalchemy import and_
+
+mod_customized = Blueprint('custom', __name__)
+
+
+@check_access_rights([Role.tester, Role.contributor, Role.admin])
+@mod_customized.before_app_request
+def before_app_request():
+    if g.user is not None:
+        g.menu_entries['custom'] = {
+            'title': 'Customize Test',
+            'icon': 'code-fork',
+            'route': 'custom.index'
+        }
+
+
+@mod_customized.route('/', methods=['GET', 'POST'])
+@login_required
+@check_access_rights([Role.tester, Role.contributor, Role.admin])
+@template_renderer()
+def index():
+    fork_test_form = TestForkForm(request.form)
+    username = fetch_username_from_token()
+    commit_options = False
+    if username is not None:
+        gh = GitHub(access_token=g.github['bot_token'])
+        repository = gh.repos(username)(g.github['repository'])
+        # Only commits since last month
+        today_date = datetime.now()
+        last_month = today_date - relativedelta(months=1)
```

Comment:
I don't think it's necessary to import another library just to have the exact month representation. Using a timedelta of 30 days would also be sufficient, and is built-in.